apps/arena/lib/arena/game_launcher.ex

```diff
@@ -6,7 +6,7 @@ defmodule Arena.GameLauncher do
   use GenServer
 
   # Amount of clients needed to start a game
-  @clients_needed 10
+  @clients_needed 2
```

Comment:
Remember to revert this

Reply:
Thanks for the reminder, forgot to restore it!